git-cliff-core/src/commit.rs

```diff
@@ -476,6 +487,24 @@ impl Serialize for Commit<'_> {
 	}
 }
 
+/// Deserialize commits into conventional commits if they are convertible
```

Comment:
Nit:
```suggestion
/// Deserialize commits into conventional commits if they are convertible.
```

Reply:
> What would be the benefit of using a separate struct?

Actually, I'm pretty new to serde so I'm not quite sure.
But, I think we don't need to manually change the number of fields when using separated struct.
We can also use proc macro directly, which is more friendly.

But those fields aren't change quite often, so I think just leave them there is also good.